odml/__init__.py

```diff
@@ -43,6 +50,7 @@ def Document(self):
 minimum_implementation = current_implementation
 
 
+#   todo: check if key is needed
```

Comment:
Does this function even work? Where is key coming from? I don't see it anywhere in the namespace.